data/siteText.yaml

```diff
@@ -1,27 +1,29 @@
 title: "## How much water do we use?"
 
 introText: |
-  ### In the map below, state size (area) is scaled proportionate to state water use. 
+  ### In the map below, State size (area) is scaled proportionally to State freshwater use. 
 
 explanationText: |
-  ### U.S. water withdrawals, 1950-present
+  ### U.S. freshwater withdrawals, 1950-present
 
   How and where Americans use water has changed with our population growth and many technological, environmental, 
   and regulatory drivers. 
   The USGS Water Availability and Use Program has documented how and where we use water for the last 65 years. 
-  Above, state size (area) is scaled proportionate to total state water use and water use by category. 
+  Above, State size (area) is scaled proportionally to total State freshwater use and freshwater use by category. 
   National water use for each year and by category are shown as bars above the timeline. Drag the timeline 
   to see data for available years, click on total or specific water-use categories, then mouse over the States 
   and the bars for water use values in million gallons of water per day (Mgal/d).
   One million gallons of water covers a football field in about 2.8 feet of water. 
   
-  Every 5 years since 1950, the USGS has compiled and estimated water-use information in cooperation with State, 
-  Federal, and local agencies to document how the Nation's water resources are used. Prior to 1960, Industrial and Thermoelectric 
-  water use categories were not assessed individually. Data at the site, county, 
-  and major watershed levels are available for some years, but not for the full period of record. Compilation of water-use data for 2015 is underway. 
-  This information is essential to accurately understand how future water demands will be met while maintaining 
-  adequate water quality and quantities for human and ecosystem needs. The USGS Water Availability and Use Program hopes to move toward even higher spatial resolution 
-  and more frequent reports in the future.
```

Comment:
Seems like by deleting all of this info we are losing the main grab/take home point of the viz - "data collected sparsely, but it's important so support it". I would rather see some more information about the why than so many details about the categories - that doesn't really interest the public which is why we refrained from getting into it (although I see how we should explain why there's not data in some of the years).